docs/updates/infromational-dec-26-2021.md

```diff
@@ -0,0 +1,89 @@
+
+During the withdraw of rewards CosmosSDK performs a sanity check, where it ensures that a validators total rewards for that period are consistent with any slashing that has occurred. Since the Gravity Bridge module was not triggering the correct hooks in the staking keeper there are no recorded slashing events.
+
+This results in the sanity check failing and an inability to withdraw rewards.
+
+This issue was not detected in our unit tests because they do not use the same initialization environment as the actual chain. We have added an integration test that validates delegating, bonding, and unbonding around Gravity Bridge slashing to prevent such an oversight in the future in addition to correcting the root cause.
+
+### Who is currently affected
+
+The following is output from a tool [located here](https://github.com/jkilpatr/inspect-delegators) designed to identify affected validators and delegators.
+
+```text
+Total 261 delegations delegators 166 bugged delegators 11 {
+    gravity1vu04rhxr4sq65vfd074m5gjzuggwukzhznzdgz,
+    gravity1kp6kn7jazzn6leqvqzdm9ftmlpp72l406fun2r,
+    gravity1qtc3axqj2pf9h92arc2lmk7keyvrzqc8y4hef3,
+    gravity12qx44c75dd8ykckexxxxjn7ygehwxkqjxyvzud,
+    gravity1gfeh4793ug8l6nu69n8lqssvkdqs8qghwxka0l,
+    gravity1xmspr43xsfu6ycjgdqqllntc62rzvncv7x8trk,
+    gravity1xf869399xq4pnxsllzvnwdcra8ly3huj9n2saq,
+    gravity19uwnjc4fhftygyzr8wv5m8m0l4kjzkpsvsmgqh,
+    gravity1d63hvdgwy64sfex2kyujd0xyfhmu7r7cedj2fg,
+    gravity104vmnec7p96qgzzdzddv5jan070v7vj6j7mslj,
+    gravity19lhajns3drveve4q8jc4kus6knjcmf2j36ta6e,
+} and bugged validators 11 {
+    gravityvaloper19lhajns3drveve4q8jc4kus6knjcmf2jq3jrsd,
+    gravityvaloper12qx44c75dd8ykckexxxxjn7ygehwxkqjh04uke,
+    gravityvaloper19uwnjc4fhftygyzr8wv5m8m0l4kjzkpsamzk2r,
+    gravityvaloper104vmnec7p96qgzzdzddv5jan070v7vj6r4zw4x,
+    gravityvaloper1gfeh4793ug8l6nu69n8lqssvkdqs8qghld0r9t,
+    gravityvaloper1kp6kn7jazzn6leqvqzdm9ftmlpp72l40tz9dqh,
+    gravityvaloper1vu04rhxr4sq65vfd074m5gjzuggwukzhncmnzk,
+    gravityvaloper1qtc3axqj2pf9h92arc2lmk7keyvrzqc847w8r9,
+    gravityvaloper1xmspr43xsfu6ycjgdqqllntc62rzvncv0d74fz,
+    gravityvaloper1xf869399xq4pnxsllzvnwdcra8ly3huj5cnwh5,
+    gravityvaloper1d63hvdgwy64sfex2kyujd0xyfhmu7r7cgxt5ru,
+} total affected stake 1803249988137
+
+```
+
+As you can see all affected delegations are self-delegations to genesis validators. Each holding `181,818 graviton` with slight
+variations based on withdraws and redelegation before their slashing and exactly how many times they have been slashed by the Gravity module.
+
+### Recommended Action
+
+The latest release version of Gravity Bridge `v1.1.0` and greater correctly set the hooks in the Gravity module `app.go`. It is important that the community upgrade to `v1.1.0` or later release or any validator that is slashed by the Gravity Bridge module will be unable to withdraw delegation rewards.
```

Comment:
It is important that the community upgrade to `v1.1.0` or later release or any validator that is slashed by the Gravity Bridge module will be unable to withdraw delegation rewards.
->
It is important that the community upgrades to release `v1.1.0` or later as any validator slashed by the Gravity Bridge module will be unable to withdraw delegation rewards. Additionally, delegators to such a validator will be unable to withdraw the funds they have delegated to that validator, along with any rewards they have earned from that delegation.